src/app/book/book.component.html

```diff
@@ -1,33 +1,23 @@
-<!-- <button type="button" class="list-group-item list-group-item-action" (click)="gotoBookDetailPage(book)">
-    <div>
-        <img class="bookCover" src="assets/imgs/bookCover.png"/>
+<div class="card flex-row">
+  <img
+    class="card-img-left example-card-img-responsive"
+    src="assets/imgs/bookCover.png"
+  />
+  <div class="card-body">
+    <button
+      id="title"
+      class="card-title h5 h4-sm"
+      (click)="gotoBookDetailPage(book)"
+    >
+      {{ book?.title }}
+    </button>
+    <h6
+      class="card-subtitle mb-2 text-muted"
+      *ngFor="let item of book?.authors"
+    >
+      {{ item }}
+    </h6>
+    <p class="card-text">{{ book?.description }}</p>
+      <app-add-copies [bookAddCopy]="book"></app-add-copies>
     </div>
```

Comment:
De knop om exemplaren toe te voegen is nu visible voor trainees

Reply:
aangepast